examples/with-wallet-stamper/src/components/turnkey-provider.tsx

```diff
@@ -139,7 +130,10 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
   }
 
   async function signInWithWallet(): Promise<User | null> {
-    const publicKey = (wallet as SolanaWalletInterface)?.recoverPublicKey();
+    const publicKey = await wallet?.getPublicKey();
+    if (!publicKey) {
+      return null;
```

Comment:
sort of a meta question but do you prefer using `null`s or `undefined`s?

Reply:
Generally I'll use `null` to distinguish between uninitialized variables and values that have been intentionally set to an empty value